src/database/standalone-sqlite.ts

```diff
@@ -2909,7 +2923,7 @@ export class StandaloneSqliteDatabase
     return this.queueWrite('bundles', 'saveDataItem', [item]);
   }
 
-  saveBundle(bundle: BundleRecord): Promise<void> {
+  async saveBundle(bundle: BundleRecord): Promise<BundleFilterIds> {
```

Comment:
nit: if you are not awaiting here you do not need the async. The return type tells caller to expect Promise.

Reply:
https://github.com/ar-io/ar-io-node/pull/289/commits/7bed4f5f8cb6345e9dc9d3b22518aae76a315cdf